packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -250,18 +266,31 @@ class _ZoomPageTransition extends StatelessWidget {
   }
 }
 
-class _ZoomEnterTransition extends StatelessWidget {
+class _ZoomEnterTransition extends StatefulWidget {
   const _ZoomEnterTransition({
     required this.animation,
     this.reverse = false,
+    required this.preferRasterization,
     this.child,
   }) : assert(animation != null),
        assert(reverse != null);
 
   final Animation<double> animation;
   final Widget? child;
+  final bool preferRasterization;
   final bool reverse;
 
+  @override
+  State<_ZoomEnterTransition> createState() => _ZoomEnterTransitionState();
+}
+
+class _ZoomEnterTransitionState extends State<_ZoomEnterTransition> with _ZoomTransitionBase {
+  // TODO(jonahwilliams): https://github.com/flutter/flutter/issues/106689
+  @override
+  bool get allowRasterization => !kIsWeb && widget.preferRasterization;
```

Comment:
This Widget does some unnecessary work in initState and didUpdateWidget when this evaluates to false - and probably also wastes some memory with unused objects...

Reply:
These are now always used that we use the raster widget fallback